mycore-user2/src/main/java/org/mycore/user2/MCRUser.java

```diff
@@ -119,12 +120,12 @@
     private MCRUser owner;
 
     /** The name of the person that this login user represents */
-    @MCRUserAttribute
+    @org.mycore.user2.annotation.MCRUserAttribute
     @XmlElement
     private String realName;
 
     /** The E-Mail address of the person that this login user represents */
-    @MCRUserAttribute
+    @org.mycore.user2.annotation.MCRUserAttribute
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.mycore.user2.annotation.MCRUserAttribute' due to existing same package import 'org.mycore.user2.*'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=4797507)